src/Files.App/Views/Layouts/ColumnsLayoutPage.xaml.cs

```diff
@@ -289,12 +289,18 @@ public void NavigateForward()
 			(ParentShellPageInstance as ModernShellPage)?.Forward_Click();
 		}
 
-		public async void NavigateUp()
+		public void NavigateUp()
 		{
 			if (ColumnHost.ActiveBlades?.Count > 1)
 				DismissOtherBlades(ColumnHost.ActiveBlades[ColumnHost.ActiveBlades.Count - 2]);
 			else
-				await Commands.NavigateUp.ExecuteAsync();
```

Comment:
Does the command need to be updated as well?

Reply:
I don't think so. The command eventually calls this `NavigateUp()` method. In other words, there would be an infinite recursive calls without this fix.